packages/@aws-cdk/core/lib/stack.ts

```diff
@@ -1373,6 +1438,47 @@ function generateExportName(stackExports: Construct, id: string) {
   return prefix + localPart.slice(Math.max(0, localPart.length - maxLength + prefix.length));
 }
 
+function isExportAList(exportedValue: any, resolved: any) {
+  const target = exportedValue.target;
+  const resolvedGetAtt = resolved['Fn::GetAtt'];
+  const resolvedRef = (resolved['Fn::Ref'] ?? resolved.Ref);
```

Comment:
Surely `Fn::Ref` isn't anything?

Reply:
`Fn::Ref` is valid CFN, but I'm not sure if CDK will ever emit it.

Reply:
TIL